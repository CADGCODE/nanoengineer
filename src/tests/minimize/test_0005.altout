======= test_0005.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
0
======= stderr =======
== structure comparison ==
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_0005.mmp  > test_0005.xyz

Total Ke = 9.315000e-20
final RMS gradient=0.971384 after 139 iterations
== structure comparison ==
at: translate (0.000000 0.000000 0.000000)
       rotate (0.000000 0.000000 0.000000)
        scale 1.000000,
standard deviation = 0.000000e+00, max delta = 0.000000e+00
======= test_0005.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_0005.mmp
# Output File: test_0005.xyz
# Trace File: test_0005.trc
# Number of Atoms: 15
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#

# rms force, high force
1 3204.68 5459.76
2 2694.06 4748.23
3 2054.62 3600.67
4 1657.38 2704.36
5 1614.53 2164.59
6 1230.45 1904.12
7 1121.54 1750.12
8 1020.79 1577.36
9 916.79 1391.91
10 826.94 1199.26
11 740.71 1090.49
12 670.87 985.95
13 611.74 864.33
14 544.29 798.97
15 496.13 722.49
16 447.00 648.96
17 397.76 570.33
18 365.30 482.86
19 324.29 448.49
20 299.29 389.61
21 257.70 362.33
22 237.16 328.56
23 215.93 299.05
24 193.28 262.48
25 170.04 230.57
26 153.41 196.96
27 139.27 177.55
28 132.77 175.52
29 110.93 142.18
30 101.23 132.40
31 92.18 117.39
32 83.52 109.75
33 79.52 107.36
34 69.96 91.40
35 64.84 85.85
36 56.14 73.34
37 52.24 67.17
38 48.13 62.16
# Switching to Conjugate-Gradient
39 35.26 0.00
40 11.99 0.00
41 8.78 0.00
42 8.76 0.00
43 8.70 0.00
44 7.93 0.00
45 6.01 0.00
46 5.99 0.00
47 5.79 0.00
48 5.66 0.00
49 5.66 0.00
50 5.66 0.00
51 5.66 0.00
52 5.66 0.00
53 5.66 0.00
54 5.66 0.00
55 5.66 0.00
56 5.66 0.00
57 5.66 0.00
58 5.66 0.00
59 5.66 0.00
60 5.66 0.00
61 5.66 0.00
62 5.65 0.00
63 5.65 0.00
64 5.65 0.00
65 5.65 0.00
66 5.65 0.00
67 5.65 0.00
68 5.65 0.00
69 5.65 0.00
70 5.65 0.00
71 5.65 0.00
72 5.64 0.00
73 5.61 0.00
74 5.57 0.00
75 5.56 0.00
76 5.55 0.00
77 5.55 0.00
78 5.53 0.00
79 5.48 0.00
80 5.40 0.00
81 5.28 0.00
82 5.16 0.00
83 5.04 0.00
84 4.83 0.00
85 4.67 0.00
86 4.57 0.00
87 4.46 0.00
88 4.34 0.00
89 4.21 0.00
90 4.02 0.00
91 3.97 0.00
92 3.81 0.00
93 3.75 0.00
94 3.64 0.00
95 3.58 0.00
96 3.46 0.00
97 3.31 0.00
98 3.18 0.00
99 3.01 0.00
100 2.92 0.00
101 2.82 0.00
102 2.75 0.00
103 2.70 0.00
104 2.62 0.00
105 2.52 0.00
106 2.47 0.00
107 2.36 0.00
108 2.35 0.00
109 2.30 0.00
110 2.24 0.00
111 2.20 0.00
112 2.13 0.00
113 2.03 0.00
114 1.92 0.00
115 1.83 0.00
116 1.78 0.00
117 1.73 0.00
118 1.71 0.00
119 1.68 0.00
120 1.62 0.00
121 1.55 0.00
122 1.50 0.00
123 1.43 0.00
124 1.42 0.00
125 1.39 0.00
126 1.37 0.00
127 1.34 0.00
128 1.31 0.00
129 1.28 0.00
130 1.23 0.00
131 1.18 0.00
132 1.13 0.00
133 1.10 0.00
134 1.07 0.00
135 1.06 0.00
136 1.03 0.00
137 1.01 0.00
138 0.97 0.00
139 0.97 nan
# Done: Minimization final rms: 0.971384, highForce: nan
======= test_0005.xyz =======
15
RMS=0.971384
C 0.026298 0.250187 -0.667958
X -0.029462 0.848762 -0.157195
X 0.013470 0.504545 -1.414576
C -1.158547 -0.735295 -0.438743
X -1.603140 -0.470945 0.155759
X -1.573781 -0.799738 -1.105366
C 1.352656 -0.525484 -0.410434
X 1.735787 -0.189963 0.190915
X 1.785943 -0.522454 -1.068635
C -0.564232 -2.125150 -0.062418
X -0.821329 -2.331220 0.653418
X -0.773656 -2.652609 -0.609160
C 0.987352 -1.992942 -0.037569
X 1.249579 -2.149352 0.687991
X 1.298065 -2.481342 -0.571032
