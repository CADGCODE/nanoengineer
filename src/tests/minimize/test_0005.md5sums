stdout 874875CF7A715579F783378315104BCB
stderr D41D8CD98F00B204E9800998ECF8427E
test_0005.xyz 1513E9798D702D5355AEBFDEFCF8F650
