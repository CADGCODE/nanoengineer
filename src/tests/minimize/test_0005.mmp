mmpformat 050130 required; 050217 optional
kelvin 300
group (Data)
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) =
(0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (0.936981, 0.308166, -0.163947, 0.014883) (5.314410) =
(0.000000, 0.000000, 0.000000) (1.000000)
datum (XY) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, =
0.000000, 1.000000) (0.000000, 1.000000, 0.000000)
datum (YZ) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (1.000000, =
0.000000, 0.000000) (0.000000, 1.000000, 0.000000)
datum (ZX) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, =
1.000000, 0.000000) (0.000000, 1.000000, 0.000000)
egroup (Data)
group (Untitled)
mol (Chunk-C1) def
atom 1 (6) (-192, 2912, 0) def
atom 2 (0) (113, 3476, 425) def
bond1 1
atom 3 (0) (-618, 3217, -564) def
bond1 1
atom 4 (6) (-1081, 2023, 889) def
bond1 1
atom 5 (0) (-1796, 2006, 602) def
bond1 4
atom 6 (0) (-1098, 2309, 1603) def
bond1 4
atom 7 (6) (935, 2061, -611) def
bond1 1
atom 8 (0) (777, 2054, -1365) def
bond1 7
atom 9 (0) (1608, 2416, -493) def
bond1 7
atom 10 (6) (-509, 593, 921) def
bond1 4
atom 11 (0) (-1056, 164, 592) def
bond1 10
atom 12 (0) (-420, 378, 1655) def
bond1 10
atom 13 (6) (1013, 469, 103) def
bond1 7 10
atom 14 (0) (1613, 391, 580) def
bond1 13
atom 15 (0) (1034, -87, -427) def
bond1 13
egroup (Untitled)
end1
group (Clipboard)
egroup (Clipboard)
end molecular machine part Untitled

