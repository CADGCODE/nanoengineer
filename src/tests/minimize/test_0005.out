======= test_0005.test =======
======= exitvalue =======
0
======= stderr =======
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_0005.mmp  > test_0005.xyz

Total Ke = 9.315000e-20
final RMS gradient=0.974089 after 144 iterations
======= test_0005.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_0005.mmp
# Output File: test_0005.xyz
# Trace File: test_0005.trc
# Number of Atoms: 15
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#

# rms force, high force
1 3003.42 4859.42
2 2558.99 4325.54
3 2196.23 3745.00
4 1859.18 3133.92
5 1508.24 2463.06
6 1246.53 1756.54
7 4693.91 10021.91
8 17569.56 37937.81
9 2820.31 6646.31
10 1167.32 2302.95
11 598.69 768.95
12 449.93 597.92
13 375.19 492.62
14 335.99 464.60
15 714.90 1581.73
16 2323.35 4776.01
17 325.60 582.16
18 169.76 220.70
19 159.86 213.99
20 147.97 194.56
21 133.19 179.92
22 133.93 205.34
23 460.51 992.75
24 922.16 1994.91
25 84.84 134.07
26 75.41 97.94
27 72.27 94.05
28 67.82 88.65
29 61.66 82.40
30 53.50 74.75
31 48.36 86.54
# Switching to Conjugate-Gradient
32 48.36 86.54
33 124.20 266.39
34 125.42 285.08
35 55.63 129.79
36 28.13 63.61
37 16.04 22.60
38 15.85 19.44
39 29.15 62.35
40 29.81 64.00
41 23.07 46.24
42 12.44 20.54
43 8.64 12.55
44 11.98 18.96
45 21.66 44.43
46 19.71 44.57
47 9.72 20.45
48 5.46 9.77
49 4.82 7.32
50 5.53 10.66
51 6.22 12.44
52 5.74 10.91
53 5.67 10.67
54 4.34 7.63
55 4.33 7.63
56 3.79 7.03
57 3.79 6.98
58 4.29 7.68
59 4.29 7.68
60 3.89 7.35
61 3.88 7.31
62 3.46 5.89
63 3.31 4.89
64 3.30 4.87
65 2.60 4.14
66 2.99 5.63
67 2.85 5.37
68 2.13 5.59
69 2.12 5.57
70 1.15 1.96
71 1.15 1.95
72 1.31 2.63
73 2.14 5.58
74 3.51 9.14
75 2.14 5.59
76 2.14 4.79
77 2.96 6.64
78 2.12 4.74
79 1.84 3.73
80 1.85 3.74
81 2.33 3.89
82 2.37 4.08
83 1.94 4.02
84 2.57 4.79
85 2.37 4.68
86 2.58 4.51
87 2.58 4.52
88 3.06 4.65
89 3.10 4.74
90 3.20 5.05
91 3.20 5.19
92 4.80 9.55
93 9.31 15.30
94 4.80 9.54
95 4.88 8.31
96 9.32 15.76
97 14.55 26.01
98 18.82 32.93
99 16.69 26.09
100 14.25 25.06
101 16.69 26.08
102 16.14 24.95
103 16.78 26.49
104 18.97 39.66
105 21.97 44.23
106 21.14 42.92
107 17.17 33.15
108 13.45 19.94
109 12.78 20.99
110 14.62 25.65
111 14.85 24.32
112 16.10 34.37
113 16.20 34.54
114 15.78 31.77
115 12.85 22.14
116 8.87 13.34
117 6.72 10.25
118 7.44 13.34
119 7.38 13.03
120 6.31 9.41
121 5.70 8.30
122 4.02 5.61
123 4.00 5.60
124 2.51 4.46
125 4.00 5.60
126 4.00 5.60
127 2.16 4.38
128 2.55 3.84
129 2.16 4.38
130 2.16 4.38
131 2.21 3.58
132 2.64 4.25
133 2.87 4.81
134 2.84 4.86
135 2.48 4.50
136 2.46 4.46
137 2.07 3.49
138 2.05 3.49
139 1.84 3.24
140 1.65 2.73
141 1.64 2.72
142 1.38 2.19
143 1.17 1.79
144 0.97 1.39
# Done: Minimization final rms: 0.974089, highForce: 1.386051
======= test_0005.xyz =======
15
RMS=0.974089
C 0.028652 0.210658 -0.722002
X -0.024267 0.867847 -0.288655
X 0.014919 0.367554 -1.495088
C -1.135725 -0.723747 -0.372584
X -1.490620 -0.460824 0.280029
X -1.632015 -0.757664 -0.983745
C 1.329708 -0.523346 -0.375641
X 1.659466 -0.180906 0.252852
X 1.807845 -0.510232 -1.001924
C -0.552317 -2.116940 -0.110335
X -0.824862 -2.385683 0.578497
X -0.743232 -2.593544 -0.708171
C 0.972423 -1.975406 -0.036614
X 1.212266 -2.136829 0.695472
X 1.302761 -2.453937 -0.567091
